float32[3]       currents_reg        #list of currents flowing in the coils [A]
float32[3]       dclink_voltages     #list of dc link voltages [V]
float32[3]       duties              #list of duty cycles for each channel (buck-stage) [0,1]
float32[3]       res_freqs           #resonant frequencies [Hz]
uint8[3]         states              #states of all channels